susemanager/bin/server-migrator.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+echo
+echo
+echo "==================================================================="
+echo "This script will migrate Uyuni server to the latest version which"
+echo "also implies replacing the underlying operating system."
+echo
+echo "During migration the services need to be shut down and after"
+echo "successful migration the server needs to be rebooted."
```

Comment:
```
echo "During migration this script will shutdown the services, and after"
echo "successful migration the server needs to be rebooted manually."
```
To make clear who is doing what :-)

Reply:
I do not understand this last comment :/

Reply:
My suggestion is to cchange lines 9 and 10 to make clear who does what.

Right now it reads
```
echo "During migration the services need to be shut down and after"
echo "successful migration the server needs to be rebooted."
```
But assuming that the services are shut down by the script I'd assume the server will be rebooted by the script as well.

So to make it clear, I'd replace that text with:
```
echo "During migration this script will shutdown the services, and after"
echo "successful migration the server needs to be rebooted manually."
```

Reply:
Ahh, now I understand. Ok, I added the "manually" even though at the end the script will tell the customer that the server needs reboot.